README.md

```diff
@@ -7,22 +7,35 @@ Installation
 ------------
 
 Just add this line to your `Gemfile`:
+
 ```ruby
 gem "solidus_auth_devise"
 ```
 
-Then run `bundle install`.
+Then, run `bundle install`.
+
+After that's done, you can install and run the necessary migrations, then seed the database:
+
+```shell
+bundle exec rake railties:install:migrations
```

Comment:
I'm not as sure about using binstubs in documentation - I think more developers will be familiar with `bundle exec`, and it's possible to generate an app without binstubs (or remove them later). Additionally, the rest of the Solidus documentation (from what I can see) uses `bundle exec`. If you feel strongly about it, however, I'll switch over.